OpenOversight/app/main/views.py

```diff
@@ -433,8 +438,19 @@ def submit_complaint():
 @main.route('/submit', methods=['GET', 'POST'])
 @limiter.limit('5/minute')
 def submit_data():
-    departments = Department.query.all()
-    return render_template('submit_deptselect.html', departments=departments)
+    # try to use preferred department if available
+    try:
+        if User.query.filter_by(id=current_user.id).one().dept_pref:
+            flash(Markup('Want to submit for another department? Change your <a href="/auth/change-dept/">default department.</a>'))
```

Comment:
Regarding the period and link, do we like

`flash(Markup('Want to submit for another department? Change your <a href="/auth/change-dept/">default department.</a>'))`

or

`flash(Markup('Want to submit for another department? Change your <a href="/auth/change-dept/">default department</a>.'))`

better?

Reply:
Also: is there a way to dismiss the flash after a few seconds? Something undocumented in flask?

Reply:
I like `flash(Markup('Want to submit for another department? Change your <a href="/auth/change-dept/">default department</a>.'))` better but I don't have strong feelings.

So I'm not aware of a feature to auto-dismiss the flashed message in Flask at least, but [this](https://pythonprogramming.net/flash-flask-tutorial/) looks like a pretty decent tutorial to make the alert dismissable.

Reply:
Just kidding, the alerts are already dismissable ;) Thanks Bootstrap.

Reply:
👍